pom.xml

```diff
@@ -47,6 +47,11 @@
       <id>placeholderapi</id>
       <url>https://repo.extendedclip.com/content/repositories/placeholderapi/</url>
     </repository>
+    <!-- GriefDefender API Repo -->
+    <repository>
+	  <id>glaremasters</id>
```

Comment:
Could you indent it the same as the other repositories? Same with the dependency below.

Reply:
Should be good now.